File: Homework_1/Homework_1(C sharp)/Homework_1(C sharp)/Form1.cs
namespace Homework_1_C_sharp_
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            btnGetItem.MouseEnter += OnMouseEnterBtn1;
            btnGetItem.MouseLeave += OnMouseLeaveBtn1;
            btnGetIndex.MouseEnter += OnMouseEnterBtn2;
            btnGetIndex.MouseLeave += OnMouseLeaveBtn2;
        }

        private void btnGetItem_Click(object sender, EventArgs e)
        {
            listBoxItem.Items.Clear();
            foreach(string s in checkedListBox.CheckedItems)
                listBoxItem.Items.Add(s);
        }

        private void btnGetIndex_Click(object sender, EventArgs e)
        {
            listBoxIndex.Items.Clear();
            for(int i=0;i<checkedListBox.CheckedIndices.Count;i++)
                listBoxIndex.Items.Add(checkedListBox.CheckedIndices[i]);
        }

        private void OnMouseEnterBtn1(object sender, EventArgs e)
        {
            btnGetItem.BackColor = Color.Green;
        }
        private void OnMouseEnterBtn2(object sender, EventArgs e)
        {
            btnGetIndex.BackColor = Color.Green;
        }
        private void OnMouseLeaveBtn1(object sender, EventArgs e)
        {
            btnGetItem.BackColor = Color.Yellow;
        }
        private void OnMouseLeaveBtn2(object sender, EventArgs e)
        {
            btnGetIndex.BackColor = Color.Yellow;
        }
    }
}

File: Homework_1/Homework_1(V.B NET)/Homework_1(V.B NET)/Form1.vb
Public Class Form1
    Private Sub BtnGetItem_Click(sender As Object, e As EventArgs) Handles BtnGetItem.Click
        ListBox1.Items.Clear()
        For Each s As String In CheckedListBox1.CheckedItems
            ListBox1.Items.Add(s)
        Next
    End Sub

    Private Sub BtnGetIndex_Click(sender As Object, e As EventArgs) Handles BtnGetIndex.Click
        ListBox2.Items.Clear()
        For i As Integer = 0 To CheckedListBox1.CheckedIndices.Count - 1
            ListBox2.Items.Add(CheckedListBox1.CheckedIndices(i))
        Next
    End Sub

    Private Sub BtnGetItem_MouseEnter(sender As Object, e As EventArgs) Handles BtnGetItem.MouseEnter
        BtnGetItem.BackColor = Color.Green
    End Sub

    Private Sub BtnGetIndex_MouseEnter(sender As Object, e As EventArgs) Handles BtnGetIndex.MouseEnter
        BtnGetIndex.BackColor = Color.Green
    End Sub

    Private Sub BtnGetItem_MouseLeave(sender As Object, e As EventArgs) Handles BtnGetItem.MouseLeave
        BtnGetItem.BackColor = Color.Yellow
    End Sub

    Private Sub BtnGetIndex_MouseLeave(sender As Object, e As EventArgs) Handles BtnGetIndex.MouseLeave
        BtnGetIndex.BackColor = Color.Yellow
    End Sub

End Class
